bootstrap.yml

```diff
@@ -16,8 +16,18 @@
       - name: load instackenv file
         include_tasks: tasks/load_instackenv.yml
 
+      - name: check if the infrared dir exists
+        stat:
+          path: "{{ item }}"
+        register: infrared_dirs
+        loop:
+          - "{{ infrared_dir }}"
+          - "{{ infrared_workspaces_dir }}"
+
       - name: setup infrared
         include_tasks: tasks/setup_infrared.yml
+        when: item.stat.exists == False
+        with_items: "{{ infrared_dirs.results }}"
 
       - name: add localhost to inventory file
```

Comment:
When you already have inventory in ~/.infrared/.workspaces/active/hosts, why you want to prepare again?